scripts/postrelease/change_management

```diff
@@ -1,23 +1,23 @@
 #!/usr/bin/env node
 
-if (!process.env.CIRCLE_TAG) {
+if (!(process.env.GITHUB_REF_TYPE === 'tag' && process.env.GITHUB_REF_NAME.startsWith('v'))) {
   console.log('Not on stable release, skipping change management trigger')
   process.exit(0)
 }
 
 const REQUEST_PROMISE = require('promise-request-retry')
 
-// Set these in CircleCI in the heroku/cli project
+// Set these in Github Actions in the heroku/cli project
 //
 const TPS_API_APP_ID = process.env.TPS_API_APP_ID
 const TPS_API_RELEASE_ACTOR_EMAIL = process.env.TPS_API_RELEASE_ACTOR_EMAIL
 const TPS_API_STAGE = process.env.TPS_API_STAGE || 'production'
 const TPS_API_TOKEN_PARAM = process.env.TPS_API_TOKEN_PARAM
 const TPS_API_URL_PARAM = process.env.TPS_API_URL_PARAM
```

Comment:
Are we sure that these env vars are making it into the script?

Reply:
@ryandagg I didn't test these ones specifically, but they are set in the env like other env vars here:
https://github.com/heroku/cli/blob/2a200f5385b5b3c8cff6a705484e009314d52470/.github/workflows/ci.yml#L210